CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 0.7.0
+
+* Relax required version of dependencies to allow gem to work with a Rails 5.1 project (PR #29)
```

Comment:
Should probably mention https://github.com/alphagov/govuk_publishing_components/pull/30 too